src/main/resources/META-INF/plugin.xml

```diff
@@ -4,7 +4,7 @@
     <vendor email="intellij-plugin-letech@aalto.fi"
             url="https://research.cs.aalto.fi/LeTech/">LeTech Group at Aalto University</vendor>
     <!--  this is the range of supported IntelliJ IDEA version the plugin can be used with  -->
-    <idea-version since-build="203"/>
+    <idea-version since-build="203.3645.34"/>
```

Comment:
oh, I'm so unsure about this ❗❗❗

Reply:
Is there a need to do this? I think it gets patched by `gradle` in its `patchPluginXml` task, so I think we should keep it at 203.

Reply:
this change is required to be able to use these two 
```java
import com.intellij.util.concurrency.annotations.RequiresReadLock;
import com.intellij.util.concurrency.annotations.RequiresWriteLock;
```

instead of deprecated `@CalledWithWriteLock` & `@CalledWithReadLock`

Reply:
Then I guess it's ok.